DataCite.js

```diff
@@ -105,9 +105,11 @@ var testCases = [
 				"seeAlso": [],
 				"attachments": [],
 				"title": "Code criminel de l'empereur Charles V vulgairement appellé la Caroline contenant les loix qui sont suivies dans les jurisdictions criminelles de l'Empire et à l'usage des conseils de guerre des troupes suisses.",
-				"url": "http://dx.doi.org/10.12763/ONA1045",
-				"DOI": "10.12763/ONA1045",
 				"date": "1734",
+				"itemID": "https://doi.org/10.12763/ona1045",
+				"abstractNote": "[object Object],[object Object],[object Object]",
```

Comment:
This is a bug. I opened an issue at https://github.com/datacite/bolognese/issues/11, you should expect a string.